src/server/conn.rs

```diff
@@ -1075,6 +1079,7 @@ pub(crate) mod spawn_all {
         NE: Into<Box<dyn StdError + Send + Sync>>,
         S: HttpService<Body, ResBody = B>,
         B: HttpBody + 'static,
+        B::Data: Send,
```

Comment:
This is on `NewSvcTask<I, N, S, E, W>`, like `NoopWatcher` it's not actually reachable, but it is used in the `NewSvcExec<I, N, S, E, W>` trait.